src/store/modules/order/actions.ts

```diff
@@ -732,6 +737,73 @@ const actions: ActionTree<OrderState , RootState> ={
     return resp;
   },
 
+  async fetchGiftCardActivationDetails({ commit }, { isDetailsPage, currentOrders }) {
```

Comment:
We should move this to OrderService as we are not mutating state using this action.